main/src/main/java/cgeo/geocaching/utils/WatchListUtils.java

```diff
@@ -0,0 +1,54 @@
+package cgeo.geocaching.utils;
+
+import android.content.Context;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import cgeo.geocaching.R;
```

Comment:
Wrong order: `cgeo` has to placed before `java`. And also before `android`.

See 
~~~
            <property name="groups" value="cgeo,android,androidx,java,javax,*" />
~~~